src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -471,6 +471,15 @@ private function getProperty(PropertyMetadata $propertyMetadata, string $propert
         if (null !== $type && !$type->isCollection() && (null !== $className = $type->getClassName()) && $this->resourceClassResolver->isResourceClass($className)) {
             $propertyData['owl:maxCardinality'] = 1;
         }
+        
+        // Add extended JSON-LD context informations
+        // @see https://api-platform.com/docs/core/extending-jsonld-context
+        $jsonldContext = $propertyMetadata->getAttributes()['jsonld_context'] ?? [];
+        foreach ($jsonldContext as $contextKey => $contextValue) {
+            if (! in_array($contextKey, ['@id', '@type'])) {
```

Comment:
only 2? Maybe `if ($contextKey === '@id' || $contextKey === '@type') { continue; }`

Reply:
You're right. I change that